docs/how-tos/protocol.rst

```diff
@@ -2,3 +2,541 @@
 
 How to define a new ``Protocol``
 ================================
+
+The **gufe** :ref:`Protocol <protocol>` system is designed as an *extensible point* of the library,
+allowing developers to write their own methods for performing free energy calculations in a form that can take full advantage of the OpenFE ecosystem.
```

Comment:
```suggestion
allowing developers to write their own methods for performing calculations in a form that can take full advantage of the OpenFE ecosystem.
```
Not all Protocols implement free energy methods - see MD protocols or Boltz-2, maybe this is explained elsewhere though.